tests/qunit/fixtures/wp-api-generated.js

```diff
@@ -10035,10 +10035,58 @@ mockedApiResponse.Schema = {
                             "maximum": 100,
                             "required": false
                         },
+                        "search": {
+                            "description": "Limit results to those matching a string.",
+                            "type": "string",
+                            "required": false
+                        },
+                        "exclude": {
+                            "description": "Ensure result set excludes specific IDs.",
+                            "type": "array",
+                            "items": {
+                                "type": "integer"
+                            },
+                            "default": [],
+                            "required": false
+                        },
+                        "include": {
+                            "description": "Limit result set to specific IDs.",
+                            "type": "array",
+                            "items": {
+                                "type": "integer"
+                            },
+                            "default": [],
+                            "required": false
+                        },
```

Comment:
This means no functionality that doesn't there before right? Have we tested?

Reply:
Oh, this is from the inheritance. Good point. 

I'll work on it. 🍺

Reply:
I've removed these inherited query params in a `get_collection_params` override - they are not currently supported for global styles revisions. If it makes sense to allow them, then maybe in a follow up.

👍🏻